content/F2015/F15-event-dropbox-coffee-chats.md

```diff
@@ -0,0 +1,18 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-05
+Category: Events
+Tags: social, industry
+Slug: dropbox-coffee-chats-f15
+Author: Anna Lorimer
+Summary: Come and grab some coffee and snacks and chat with some female engineers from Dropbox!
+
+Join us and several female engineers from Dropbox for coffee and snacks at William's in the plaza.
+
+Note: This is a [women-only](http://wics.uwaterloo.ca/faq/) event.
```

Comment:
As I mentioned in #222, the links never should have been absolute. It breaks the development version of the site (e.g. the Heroku one). This got added (and wasn't caught) in like S15 and has been cargo culted ever since.

(Cargo-culting: copy-pasting something blindly, such as a config file or piece of text.)